src/styles/components/_hawk-autocorrect-suggestion.scss

```
@@ -1,13 +1,295 @@
+			cursor: pointer;
+		}
+
+		&:focus,
+		&:hover {
+			border: 1px solid $primary-color;
+			background: white;
+		}
+	}
+
+	.image-search-btn {
+		//display: flex;
```

Stylelint (17.5.0)

[error] Expected a space after // (scss/double-slash-comment-whitespace-inside)

(scss/double-slash-comment-whitespace-inside)